pyrasterframes/src/main/python/docs/raster-catalogs.pymd

```diff
@@ -87,20 +87,20 @@ scene_list = spark.read \
     .format("csv") \
     .option("header", "true") \
     .load(SparkFiles.get("2018-07-04_scenes.txt"))
-scene_list.head(3)
+scene_list.show(4, truncate=False)
 ```
 
 Observe the scenes list file has URIs to `index.html` files in the download_url column. The image URI's are in the same directory. The filenames are of the form `${gid}_B${band}.TIF`. The next code chunk builds these URIs, which completes our catalog.
 
-```python
+```python, show_remote_catalog, results='raw'
 modis_catalog = scene_list \
     .withColumn('base_url',
         F.concat(F.regexp_replace('download_url', 'index.html$', ''), 'gid',)
     ) \
     .withColumn('B01' , F.concat('base_url', F.lit("_B01.TIF"))) \
     .withColumn('B02' , F.concat('base_url', F.lit("_B02.TIF"))) \
     .withColumn('B03' , F.concat('base_url', F.lit("_B03.TIF")))
-modis_catalog.head(3)
+modis_catalog.show(4, truncate=True)
```

Comment:
this table is very wide, not responsive, and no scroll bar... not sure what we can do about it. Maybe some css stuff?

Reply:
Tweaked CSS to break anchors on any character.